### PR TITLE
Comparison operators

### PR DESCRIPTION
**Pull request by ElenaLadutska**
Added 6.08 and 8.08 deadline files

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+// First task
+
+// const a = prompt('Text a number, please');
+
+// if(a == 10){
```
Comment (kathrynowy) [inline index.js:5]: use strict comparison

File: index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+//   alert('True');
+// }else{
+//   alert('False');
+// };
```
Comment (kathrynowy) [inline index.js:9]: write it in 1 line please

File: index.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+//     alert('First quarter');
+//   }else if(min > 15 && min <= 30){
+//     alert('Second quarter');
+//   }else if(min > 30 && min <= 45){
+//     alert('Third quarter');
+//   }else if(min > 45 && min <= 59){
+//     alert('Fouth quarter');
+//   }else if(min > 59){
+//     alert('Number out of range. Enter a number from 0 to 59, please');
+//   }else if(isNaN(min) === true){
```
Comment (kathrynowy) [inline index.js:27]: do you need here === true?

File: index.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+// };
+
+
+//Third task
+
+// const a = 1;
+// const a = 0;
+// const a = -3;
+
+// if(a == 0){
```
Comment (kathrynowy) [inline index.js:41]: use strict comparison

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ElenaLadutska) [reply index.js:5]: fixed

Comment (ElenaLadutska) [reply index.js:5]: const a = +prompt('Text a number, please');
(a === 10) ? alert('true') : alert('false');

File: index.js (hunk 2 above)
Comment (ElenaLadutska) [reply index.js:9]: fixed

File: index.js (hunk 3 above)
Comment (ElenaLadutska) [reply index.js:27]: no. deleted. fixed

File: index.js (hunk 4 above)
Comment (ElenaLadutska) [reply index.js:41]: fixed
(a === 0) ? alert('true') : alert('false');

**Pull request merged**